Bitcell symmetry

bkoppelmann opened the pull request:
Hi Matt,
I made the first step to support bitcells, that also need to be mirrored on the y axis (i.e. thin cells). I added the option to technologies, to specify which mirroring they want. Then the placement/routing takes care of the rest. 
Cheers,
Bastian

bkoppelmann commented:
Rebased onto dev.

mguthaus commented:
I like the new cell properties class. I noticed there is a missing import in the scn4m_subm tech file though:
from custom_cell_properties import CellProperties

After this, I get some errors in the new abstraction of the bitcell array. 
TypeError: __init__() missing 1 required positional argument: 'column_offset'

bkoppelmann commented:
Huh? I thought I already fixed that. On my side the [diff](https://github.com/VLSIDA/OpenRAM/pull/64/files#diff-39038f1d5030ea16eabbc548abc98a10) looks correct. Maybe some github shenanigans? I think I also fixed the missing argument in init(). If you still have the problem, can you send me a reproducer?

mguthaus commented:
I think I might have run it with OPENRAM_HOME incorrectly set. It looks like it is running now.
